CHANGELOG.md

```diff
@@ -13,6 +13,14 @@
 
 # v1.91.1
 
+## Enhancements
+
+* Updates endpoint for deleting stack from `POST` to `DELETE` and removing `/delete` from endpoint by @sahar-azizighannad  [#1205](https://github.com/hashicorp/go-tfe/pull/1205)
+* Updates endpoint for force deleting stack from `POST` to `DELETE` and removing `/force-delete` from endpoint and adding `?force=true` by @sahar-azizighannad  [#1205](https://github.com/hashicorp/go-tfe/pull/1205)
```

Comment:
I think what Chris meant in [his comment here](https://github.com/hashicorp/go-tfe/pull/1205#discussion_r2316736614) didn't pertain to the change being made but rather it being called out in the CHANGELOG. Since updating the routes isn't a user facing change, we can omit these entries.